cypress.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+    video: false,
+    retries: {
+        runMode: 2,
+    },
+    component: {
+        setupNodeEvents() {
+            return require('./cypress/plugins/index.ts');
+        },
+        viewportHeight: 1280,
+        viewportWidth: 800,
+        specPattern: ['./src//**/*.cy.tsx', './**/*.spec.ts', './src//**/*.spec.tsx'],
```

Comment:
/src// the 2 // are intented?

Reply:
oh, no - thanks. Adapted in [ced93cc](https://github.com/Frontify/fondue/pull/881/commits/ced93cc13033c3e3660257880031d56da99aa50d)